data/config.yaml

```diff
@@ -1,8 +1,8 @@
 _id: d779730c7cc94155877a24e2e8be2ad1
-created: '2022-07-26T12:13:07.493686'
+created: '2022-04-18T14:36:30.754421'
```

Comment:
Not this PR, but we really need to work out what keeps changing this and fix it. @dangunter @lbianchi-lbl

Reply:
I think its running test that include something data related.

Reply:
There are two DMF tests that create new config.yaml files, it's possible one or both are not actually creating temporary directories as expected: [test_datasets](https://github.com/IDAES/idaes-pse/blob/main/idaes/core/dmf/tests/test_datasets.py) [test_dmfbase](https://github.com/IDAES/idaes-pse/blob/main/idaes/core/dmf/tests/test_dmfbase.py)